generators/app/templates/src/components/Spinner/index.tsx

```diff
@@ -13,7 +13,7 @@ interface SpinnerConfig extends SpinnerProps {
   classNameContainer?: string;
 }
 
-export const withSpinner = (spinnerConfig: SpinnerConfig = {}) => <P extends object>(
+export const withSpinner = (spinnerConfig: SpinnerConfig = {}) => <P extends {}>(
```

Comment:
Isn't this against one of the typescript rules? the {} I mean

Reply:
Yeah, we could ban that and use another alternative.
I left it this way because, it's the way react and other libs type props.
ref: https://github.com/typescript-eslint/typescript-eslint/issues/2063